libs/ledger-live-common/src/device-core/managerApi/use-cases/getLatestFirmwareForDevice.test.ts

```diff
@@ -39,4 +38,104 @@ describe("getLatestFirmwareForDevice", () => {
     // then
     expect(response).rejects.toBeInstanceOf(UnknownMCU);
   });
+
+  test("Device not in device update mode with matching MCUs", async () => {
```

Comment:
I feel like these test names are not very clear, and it can be improved largely by using `it` and following a format like this one: `it("should return this when blablabla")`
![Screenshot 2024-02-12 at 18 04 57](https://github.com/LedgerHQ/ledger-live/assets/91890529/7fb796b2-a055-4d3e-a9b9-81d9f9cde6e0)